TRANSLATING.md

```diff
@@ -115,3 +116,49 @@ The change immediately goes live for all languages at the same time.
 Some pull requests do not require a translation.
 They are labeled with “not translatable”.
 You can safely ignore them.
+
+## Tips & Tricks
+
+### Creating the Content Files
+
+You can basically copy the English docs from `site/docs/` to a new directory next to the other languages, and then translating every page.
+
+### Creating the Configuration Files
+
+There are several places in the configuration that need to be translated, too.
+It is best to go over every file in `site/docs/.vitepress/` and check if there are things that need to be adjusted for your language.
+Regarding the config, it may make sense not to copy the English version, as there are some exceptions to English in how it is configured.
+Instead, you can copy a different language for the structure (and read the English config for the content).
+For example, it helps to display three files next to each other: Spanish for structure, English for content, and your new language.
+
+### Freedom in Deviating From English
+
+> **TL;DR:** Keep the same structure and content, deliver it in your own language.
+
+You have quite some freedom in how you want to express the content in your own language.
+You know your language best, and we are not going to tell you how to phrase things, or where to split up your sentences.
```

Comment:
@niusia-ua I tried to say this here. How can it be improved?

Reply:
Wait, has it always been here? If so, then I just didn't pay attention. Then there is nothing to be changed. It's good enough.

Reply:
Yep :D
Awesome!